content/docs/command-reference/params/index.md

```diff
@@ -18,44 +18,44 @@ positional arguments:
 
 In order to track parameters and hyperparameters associated to machine learning
 experiments in <abbr>DVC projects</abbr>, DVC provides a different type of
-dependencies: _parameters_. Parameters are defined using the the `-p`
-(`--params`) option of `dvc run`, using simple names like `epochs`,
-`learning-rate`, `batch_size`, etc.
+dependencies: _parameters_. Params are defined in the `params` field of
+`dvc.yaml` or using the the `-p` (`--params`) option of `dvc run`. They usually
+have simple names like `epochs`, `learning-rate`, `batch_size`, etc.
```

Comment:
better to move this sentence up

Reply:
Do you mean "They usually have simple names ..."? That looks like the least relevant sentence, it's a list of examples.

No strong opinion but what do you mean by move up? Up one-over? Start the paragraph with it?